src/OmniSharp.Host/OmniSharpWorkspaceInitializer.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Composition.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OmniSharp.Options;
+using OmniSharp.Roslyn;
+using OmniSharp.Roslyn.Options;
+using OmniSharp.Services;
+
+namespace OmniSharp
+{
+    public class OmniSharpWorkspaceInitializer
```

Comment:
Why does everything need to be prefixed with `OmniSharp`? We're in the `OmniSharp` namespace!